src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+class FigureSupplier {
+    private static final Random random = new Random();
+    private static final double MAX_SIZE = 100.0;
```

Comment:
Consider defining a constant for the number of figure types (5 in this case) to avoid using magic numbers. This will improve code readability and maintainability.